src/api/app/models/obs_factory/distribution_strategy_factory.rb

```diff
@@ -114,8 +112,8 @@ def totest_version
     # @return [String] version string
     def published_version
       begin
-        f = open(repo_url)
-      rescue ::OpenURI::HTTPError
+        f = URI.parse(repo_url)
+      rescue ::OpenURI::HTTPError => e
```

Comment:
and I guess it's not thowing OpenURI exceptions either?

Reply:
It also does not need the `e` variable. I just wanted to add the commit and check if the test suite passes.
But that one is currently broken :cry:

Reply:
well, the rspec test suite on travis passed - so it's broken? :)

Reply:
What is broken? The only failing travis tests I see are in the api test suite, not rspec.

And since they all seem to be related with publishing I suspect it's the createrepo_c problem we have in master.